.github/workflows/test.yml

```diff
@@ -0,0 +1,48 @@
+name: Django Birdbath CI
+
+on:
+  push:
+    branches:
+      - main
+      - 'stable/**'
+
+  pull_request:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+permissions:
+  contents: read # to fetch code (actions/checkout)
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - name: Set up Python 3.9
+        uses: actions/setup-python@v5
+        with:
+          python-version: '3.9'
```

Comment:
suggestion for follow up PR: contrary to popular belief, it is best to use the latest stable/tested version (so 3.12) to benefit from improvements. All current tooling will run on a newer version but do the linting for prev versions

Reply:
That makes more sense, thanks for explaining it.